.travis.yml

```diff
@@ -1,6 +1,31 @@
 language: java
 jdk: openjdk8
 
+jobs:
+  include:
+    - stage: test
+      name: "Unit tests"
+      script:
+      - "./gradlew test"
+    - stage: compat-test-gradle-4
+      name: "Stutter compat test for gradle 4.9 version"
+      script: "./gradlew compatTest4.9"
+
+    - script: "./gradlew compatTest4.10.3"
+      name: "Stutter compat test for gradle 4.10.3 version"
+    - stage: compat-test-gradle-5
+      name: "Stutter compat test for gradle 5.1.1 version"
+      script: "./gradlew compatTest5.1.1"
+
+    - script: "./gradlew compatTest5.6.4"
+      name: "Stutter compat test for gradle 5.6.4 version"
+    - stage: compat-test-gradle-6
+      name: "Stutter compat test for gradle 6.0.1 version"
+      script: "./gradlew compatTest6.0.1"
+
+    - script: "./gradlew compatTest6.5"
+      name: "Stutter compat test for gradle 6.5 version"
+
```

Comment:
I'm finding this a bit confusing to understand. Does it define stages for each individual Gradle major version and within each stage it has multiple `script`s for each minor version?

Reply:
It looked a bit confusing at the beginning for me too, but at the end what it will do is:

- Create a stage for the tests.
- Create a stage for each gradle major version (4, 5 and 6)
- Run the compatTest compat test of each 'minor' version in parallel.

More info: https://docs.travis-ci.com/user/build-stages/#naming-your-jobs-within-build-stages

Reply:
I just pushed a new commit making a change in the line breaks, I hope that makes it a bit more readable :)